rpcserver.go

```diff
@@ -2104,6 +2104,11 @@ func handleGetNetworkHashPS(s *rpcServer, cmd interface{}, closeChan <-chan stru
 		endHeight = best.Height
 	}
 
+	// Calculate the number of blocks per retarget interval based on the
+	// chain parameters.
+	blocksPerRetarget := int32(s.server.chainParams.TargetTimespan /
+		s.server.chainParams.TargetTimePerBlock)
+
```

Comment:
It's based on constants, so it's probably not a big deal that it's calculating it here.  However, I don't imagine it would hurt to expose it on the chain instance either.